src/kfflang/forge/kotlin/thedarkcolour/kotlinforforge/AutoKotlinEventBusSubscriber.kt

```diff
@@ -132,9 +133,9 @@ public object AutoKotlinEventBusSubscriber {
 
     private fun registerTo(any: Any, target: Mod.EventBusSubscriber.Bus, mod: KotlinModContainer) {
         if (target == Mod.EventBusSubscriber.Bus.FORGE) {
-            target.bus().get().register(any)
+            target.bus().get()?.register(MethodHandles.lookup(), any)
```

Comment:
Why is the bus supplier now nullable? Should this log an error if the bus is null?